src/main/java/com/icthh/xm/uaa/config/ApplicationProperties.java

```diff
@@ -41,6 +41,9 @@ public class ApplicationProperties {
     private Set<String> clientGrantTypes;
     private Set<String> clientScope;
     private Set<String> defaultClientId;
+    private String accessTokenUrl;
```

Comment:
I think accessTokenUrl can be hardcoded as it depends on spring, but as you wish

Reply:
I agree with @pkev777